sktime/transformers/series_as_features/signature_based/transformers.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.pipeline import Pipeline
+
+
+def get_augmentation_pipeline(aug_list):
+    """Buids an sklearn pipeline of augmentations from a list of strings.
+
+    Parameters
+    ----------
+    aug_list: list of strings
+              A list of strings that determine the augmentations to apply, and in which order to apply them (the first
+              string will be applied first). Possible augmentation strings are ['leadlag', 'ir', 'addtime', 'cumsum',
+              'basepoint']
+
+    Returns
+    -------
+    sklearn.Pipeline
+        The transforms, in order, as an sklearn pipeline.
+
+    Examples
+    --------
+    >>> get_augmentation_pipeline(['leadlag', 'ir', 'addtime'])
+    Pipeline([
+        ('leadlag', LeadLag()),
+        ('ir', InvisibilityReset()),
+        ('addtime', AddTime())
+    ])
+    """
+    # Dictionary of augmentations
+    AUGMENTATIONS = {
+        'leadlag': LeadLag(),
+        'ir': InvisibilityReset(),
+        'addtime': AddTime(),
+        'cumsum': CumulativeSum(),
+        'basepoint': Basepoint()
+    }
+
+    pipeline = Pipeline([
+        (tfm_str, AUGMENTATIONS[tfm_str]) for tfm_str in aug_list
+    ])
+
+    return pipeline
+
+
+class AddTime(BaseEstimator, TransformerMixin):
+    """Add time component to each path.
+
+    For a path of shape [N, L, C] this adds a time channel of length L running in [0, 1]. The output shape is thus
+    [N, L, C + 1] with time being the first channel.
+    """
+    def fit(self, data, labels=None):
+        return self
+
+    def transform(self, data):
```

Comment:
if these are public transformer, you need to add input checks from `sktime/utils/validation/series_as_features.py` and update the `self._is_fitted` flag in fit or use the default `fit` implementation in `BaseSeriesAsFeaturesTransformer`, otherwise I'd suggest to add a leading underscore to the class name